x-pack/plugins/enterprise_search/public/applications/shared/licensing/manage_license_button.test.tsx

```diff
@@ -35,7 +37,7 @@ describe('ManageLicenseButton', () => {
       const wrapper = shallow(<ManageLicenseButton />);
 
       expect(wrapper.find(EuiButton).prop('href')).toEqual(
-        expect.stringContaining('/license-management.html')
+        expect.stringContaining(`${docLinks.licenseManagement}`)
```

Comment:
```suggestion
        expect.stringContaining(docLinks.licenseManagement)
```
No need for a template literal here